<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from /home/mcube/Documents/tools_ws/src/yumi/yumi_description/urdf/yumi.srdf.xacro | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="yumi" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <group name="left">
    <chain base_link="yumi_body" tip_link="yumi_link_7_l"/>
  </group>
  <group name="left_hand">
    <link name="gripper_l_base"/>
    <link name="gripper_l_finger_r"/>
    <link name="gripper_l_finger_l"/>
  </group>
  <end_effector group="left_hand" name="left_hand" parent_group="left" parent_link="yumi_link_7_l"/>
  <!-- Disable collisions between hand links -->
  <disable_collisions link1="gripper_l_base" link2="gripper_l_finger_r"/>
  <disable_collisions link1="gripper_l_base" link2="gripper_l_finger_l"/>
  <disable_collisions link1="gripper_l_finger_r" link2="gripper_l_finger_l"/>
  <!-- Disable arm self-collisions -->
  <!-- TODO -->
  <group name="right">
    <chain base_link="yumi_body" tip_link="yumi_link_7_r"/>
  </group>
  <group name="right_hand">
    <link name="gripper_r_base"/>
    <link name="gripper_r_finger_r"/>
    <link name="gripper_r_finger_l"/>
  </group>
  <end_effector group="right_hand" name="right_hand" parent_group="right" parent_link="yumi_link_7_r"/>
  <!-- Disable collisions between hand links -->
  <disable_collisions link1="gripper_r_base" link2="gripper_r_finger_r"/>
  <disable_collisions link1="gripper_r_base" link2="gripper_r_finger_l"/>
  <disable_collisions link1="gripper_r_finger_r" link2="gripper_r_finger_l"/>
  <!-- Disable arm self-collisions -->
  <!-- TODO -->
</robot>
